woocommerce-services.php

```diff
@@ -267,22 +267,18 @@ public static function plugin_deactivation() {
 			require_once __DIR__ . '/classes/class-wc-connect-tracks.php';
 			require_once __DIR__ . '/classes/class-wc-connect-wcst-to-wcshipping-migration-state-enum.php';
 
-			$migration_state = get_option( 'wcshipping_migration_state' );
-			if ( ! $migration_state || intval( $migration_state ) !== WC_Connect_WCST_To_WCShipping_Migration_State_Enum::COMPLETED ) {
```

Comment:
👋 Hey @Ferdev, this was introduced in https://github.com/Automattic/woocommerce-services/pull/2765. 

This PR no longer needs to rely on the `plugin_deactivation()` to update the `wcshipping_migration_state` flag to "completed". When the migration modal finishes and redirects the users to `http://localhost/wp-admin/plugins.php`, the `wcshipping_migration_state` will be set to `WC_Connect_WCST_To_WCShipping_Migration_State_Enum::COMPLETED`.

I changed the condition for the tracking to "if it is completed, then do tracking". Let me know if this doesn't work.